detox/src/devices/ios/AppleSimUtils.js

```diff
@@ -70,8 +71,9 @@ class AppleSimUtils {
     const isBooted = await this.isBooted(udid);
 
     if (!isBooted) {
+      const userLaunchArgs = (argparse.getArgValue('deviceLaunchArgs') || '');
```

Comment:
Why not call the variable `deviceLaunchArgs`, as is?

Reply:
well then I suppose the outcome would be `boot UDID undefined`, wouldn't it?

Reply:
but I meant rename in two places, no?